### DISCUSSION TITLE
Had anybody tried NX with msbuild?

### DISCUSSION THREAD

**Discussion by ManikantaVenkateshappa** [nrwl/nx#15727] [General]
Hello, 
We are integrating NX with our existing MSBuild mammoth scripts. I was wondering if anybody already did such a work or any pointers in this direction would be much appreciated. 
Thanks in advance

-Kind Regards
Manikanta

**Comment by AgentEnder** [comment]
Not MSBuild, but @nx-dotnet/core uses the .NET CLI, you could probably take some inspiration from it.
